### PR TITLE
Clarification on late competitors to MBLD attempts

### PR DESCRIPTION
**Pull request by Ivan-Brigidano**
Based on Board feedback.

### CODE REVIEW

File: wca-guidelines.md [hunk 1]
```diff
@@ -211,6 +211,7 @@ To be more informative, each Guideline is classified using one of the following
 ## <article-H><multiple-blindfolded><multipleblindfoldedsolving> Article H: Multi-Blind Solving
 
 - H1+) [ADDITION] If a puzzle with a duplicate scramble is found during an attempt, the puzzle may be re-scrambled using a different scramble sequence, at the discretion of the WCA Delegate.
+- H1b+) ADDITION A competitor may join a Mulyti-Blind attempt that is already in progress, at the discretion of the WCA Delegate. In this case, their time limit is the time remaining (i.e. they must finish when other competitors reach their 60 minute time limit). Exception: a competitor must not be permitted to join a Multi_blind attempt if any competitor has finished their attempt already, or if there is any other risk that information about the scramble may have become known to the competitor who wishes to join.
```
Comment (Ivan-Brigidano) [inline wca-guidelines.md]:
```suggestion
- H1b+) [ADDITION] A competitor may join a Multi-Blind attempt that is already in progress, at the discretion of the WCA Delegate. In this case, their time limit is the time remaining (i.e. they must finish when other competitors reach their 60 minute time limit). Exception: a competitor must not be permitted to join a Multi-Blind attempt if any competitor has finished their attempt already, or if there is any other risk that information about the scramble may have become known to the competitor who wishes to join.
```

**Comment by xsrvmy** [comment]
Two problems with the wording:
1. It does not address original time limits that are under 60 minutes
2. What if a competitor tries to take advantage of the wording by getting a fast DNF to disqualify other competitors (in a bo2/3 round)?

**Comment by Ivan-Brigidano** [comment]
> It does not address original time limits that are under 60 minutes

What do you mean by that?

> What if a competitor tries to take advantage of the wording by getting a fast DNF to disqualify other competitors (in a bo2/3 round)?

If this ever happen, the WCA Delegate should consider disqualifying the competitor (see [Regulation 2k6](https://www.worldcubeassociation.org/regulations/#2k6)).

**Comment by Jambrose777** [comment]
> > It does not address original time limits that are under 60 minutes
> 
> What do you mean by that?
> 
> > What if a competitor tries to take advantage of the wording by getting a fast DNF to disqualify other competitors (in a bo2/3 round)?
> 
> If this ever happen, the WCA Delegate should consider disqualifying the competitor (see [Regulation 2k6](https://www.worldcubeassociation.org/regulations/#2k6)).

I should disqualify someone for attempting a quick 2/2 after a large attempt, like a lot of people do? That does not sound like a great thing to recommend to Delegates.

**Comment by Ivan-Brigidano** [comment]
> I should disqualify someone for attempting a quick 2/2 after a large attempt, like a lot of people do? That does not sound like a great thing to recommend to Delegates.

This is not what I'm trying to say. I mean that a competitor could be disqualified if attempting an intentionally poor result to disqualify other competitors.

**Review by hanwu85 — APPROVED** [review]

**Comment by xsrvmy** [comment]
For the first comment, I am refering to a competitor attempting less than 6 cubes. The current wording may increase the time limit in this case.

File: wca-guidelines.md [hunk 2]
```diff
@@ -211,6 +211,7 @@ To be more informative, each Guideline is classified using one of the following
 ## <article-H><multiple-blindfolded><multipleblindfoldedsolving> Article H: Multi-Blind Solving
 
 - H1+) [ADDITION] If a puzzle with a duplicate scramble is found during an attempt, the puzzle may be re-scrambled using a different scramble sequence, at the discretion of the WCA Delegate.
+- H1b+) [ADDITION] A competitor may join a Multi-Blind attempt that is already in progress, at the discretion of the WCA Delegate. In this case, their time limit is the time remaining (i.e. they must finish when other competitors reach their 60 minute time limit). Exception: a competitor must not be permitted to join a Multi-Blind attempt if any competitor has finished their attempt already, or if there is any other risk that information about the scramble may have become known to the competitor who wishes to join.
```
Comment (Ivan-Brigidano) [inline wca-guidelines.md]:
```suggestion
- H1b+) [ADDITION] A competitor may join a Multi-Blind attempt that is already in progress, at the discretion of the WCA Delegate. In this case, their time limit is the time remaining (i.e. they must finish when other competitors reach their time limit). Exception: a competitor must not be permitted to join a Multi-Blind attempt if any competitor has finished their attempt already, or if there is any other risk that information about the scramble may have become known to the competitor who wishes to join.
```

**Pull request merged**